docs/acsengine.md

```diff
@@ -97,18 +97,18 @@ The build process leaves the compiled `acs-engine` binary in the `bin` directory
 
 ```
 # ./bin/acs-engine
-ACS-Engine deploys and manages Kubernetes, Swarm Mode, and DC/OS clusters in Azure
+ACS-Engine deploys and manages Kubernetes, OpenShift, Swarm Mode, and DC/OS clusters in Azure
```

Comment:
This wasn't added in this PR, just reflects the current state of the acs-engine command (only changed docs to match output of command)